tools/breaking-changes/clone.sh

```diff
@@ -0,0 +1,10 @@
+mkdir -p src
+git clone git@github.com:SAP/spartacus.git ./src/old
```

Comment:
I have to run it: `git clone https://github.com/SAP/spartacus.git ./src/old`

Reply:
I run this `clone.sh`, and got `Permission denied`.

Reply:
I can't see why.  Could git have messed up the exec flag?
if you run `ls -l` , can you see the exec flag?
If you run `chmod 744 clone.sh` does it fix it?

Reply:
in the end this file is half documentation, half convenience.  There really are just 2 commands in there.